sdk/daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslatorSpec.scala

```diff
@@ -136,11 +134,6 @@ class ValueTranslatorSpec(majorLanguageVersion: LanguageMajorVersion)
         SVariant("Mod:Either", "Right", 1, SText("some test")),
       ),
       (Ast.TTyCon("Mod:Color"), ValueEnum("", "blue"), SEnum("Mod:Color", "blue", 2)),
-      (
-        Ast.TApp(Ast.TTyCon("Mod:Tricky"), Ast.TBuiltin(Ast.BTList)),
-        ValueRecord("", ImmArray("" -> ValueNil)),
-        SRecord("Mod:Tricky", ImmArray("x"), ArrayList(SValue.EmptyList)),
-      ),
```

Comment:
Since Tricky is not serializable, it should not be handle the ValueTranslator.
Cannot remember why it was added.